src/internal/packager/validate/validate.go

```diff
@@ -18,6 +18,10 @@ import (
 
 // Run performs config validations
 func Run(pkg types.ZarfPackage) error {
+	if pkg.Kind == "ZarfInitConfig" && pkg.Metadata.YOLO {
+		return fmt.Errorf("sorry, you can't YOLO an init package")
```

Comment:
This seems like something that would be good for the `english.go` lang file. Low enough hanging fruit.